internal/flowcontrol/base_flow_controller.go

```diff
@@ -60,10 +60,8 @@ func (c *baseFlowController) sendWindowSize() protocol.ByteCount {
 	return c.sendWindow - c.bytesSent
 }
 
+// needs to be called with locked mutex
```

Comment:
It shouldn't be a public fn then, no?

Reply:
That probably applies to all function of the `baseFlowController`.

Reply:
Done.